src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs

```diff
@@ -563,6 +563,9 @@ public double Y
             }
         }
 
+        internal double ActualHeight { get; set; }
```

Comment:
Just curious, can you explain why these were needed?

Reply:
These were needed to get the exact height and width from the node view and exposed to the portviewmodel via the nodeviewmodel. The width and height are needed to compute the popup placement positioning.